packages/react-tooltip/src/components/Tooltip/useTooltipStyles.ts

```diff
@@ -0,0 +1,84 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
+import { TooltipState } from './Tooltip.types';
+import { Theme } from '@fluentui/react-theme';
+
+/**
+ * The height of the triangle used for the arrow that points at the tooltip's target
+ */
+export const arrowHeight = 6;
+
+export const tooltipBorderRadius = (theme: Theme) => theme.global.borderRadius.medium;
+
+/**
+ * Styles for the root slot
+ */
+const useStyles = makeStyles({
+  root: theme => ({
+    display: 'none',
+    padding: '5px 12px 7px 12px',
+    maxWidth: '240px',
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.base[200],
+    lineHeight: theme.global.type.lineHeights.base[200],
+    borderRadius: tooltipBorderRadius(theme),
+
+    background: theme.alias.color.neutral.neutralForeground2, // TODO should be neutralBackgroundInverted
+    color: theme.alias.color.neutral.neutralForegroundInverted,
+
+    // TODO need to add versions of theme.alias.shadow.shadow8, etc. that work with filter
+    filter:
+      `drop-shadow(0 0 2px ${theme.alias.color.neutral.neutralShadowAmbient}) ` +
+      `drop-shadow(0 4px 8px ${theme.alias.color.neutral.neutralShadowKey})`,
+
+    // Popper sets data-popper-placement on the root element, which is used to align the arrow
+    '&[data-popper-placement^="top"] > .tooltip-arrow': { bottom: 0, '--angle': '0' },
+    '&[data-popper-placement^="right"] > .tooltip-arrow': { left: 0, '--angle': '90deg' },
+    '&[data-popper-placement^="bottom"] > .tooltip-arrow': { top: 0, '--angle': '180deg' },
+    '&[data-popper-placement^="left"] > .tooltip-arrow': { right: 0, '--angle': '270deg' },
```

Comment:
I suggest use `:global` on `arrow` instead:

```ts
    'global([data-popper-placement^="top"])': { bottom: 0, '--angle': '0' },
    'global([data-popper-placement^="right"])': { left: 0, '--angle': '90deg' },
    'global([data-popper-placement^="bottom"])': { top: 0, '--angle': '180deg' },
    'global([data-popper-placement^="left"])': { right: 0, '--angle': '270deg' },
```

In this case we will avoid slot coupling as we will not use `> .tooltip-arrow` selector.

Reply:
I'm not familiar with `:global`. Would it cause the styling get applied to other components besides Tooltip that use popper? (`data-popper-placement` is set by Popper and would be on any popup)

Reply:
```ts
{
  ':global([data-popper-placement^="top"])': { bottom: 0 },
  ':global([data-popper-placement^="right"])': { left: 0 },
}
```

Will produce:

```css
[data-popper-placement^="top"] .fotnskf0 {
  bottom: 0;
}
[data-popper-placement^="right"] .f1nq055x {
  left: 0;
}
```

There can be a collision only if multiple components in a tree will define `data-popper-placement`, but as we always render `Tooltip`s into portals, it should be safe.